src/manager/agent.manager.service.ts

```diff
@@ -73,7 +73,7 @@ export class AgentManagerService {
         // @tothink move this caching to db
         // adding one second to cache record timeout so that spinDownAgent has time to process before cache deletes the record
         Logger.info(`record cache limit set to: ${(ttl === 0 ? ttl : ttl + 1)}`);
-        await this.cache.set(agentId, { containerId, adminPort, httpPort, adminApiKey, ttl }, {ttl: (ttl === 0 ? ttl : ttl + 1)});
+        await this.cache.set(agentId, { containerId, adminApiKey, ttl }, {ttl: (ttl === 0 ? ttl : ttl + 1)});
```

Comment:
We should be able to remove this .set entirely

Reply:
We need to cache adminAPIKey for the governance Policy handlers.  Unfortunately we cannot remove setting and removing cache items